Tests/Helpers/Operators.swift

```diff
@@ -0,0 +1,8 @@
+infix operator • {
+    associativity right
+    precedence 170
+}
+
+func • <A, B, C> (f: B -> C, g: A -> B) -> A -> C {
+    return { x in f(g(x)) }
```

Comment:
Variable Name Min Length Rule Violation: Variable name should be 2 characters or more: currently 1 characters (variable_name_min_length)